shared/actions/teams/index.js

```diff
@@ -443,6 +476,65 @@ function* _setupTeamHandlers(): SagaGenerator<any, any> {
   })
 }
 
+function* _updateTopic({
+  payload: {conversationIDKey, newTopic},
+}: Constants.UpdateTopic): SagaGenerator<any, any> {
+  const state: TypedState = yield select()
+  const path = pathSelector(state)
+  const teamname = Constants.getTeamNameFromConvID(state, conversationIDKey) || ''
+  const param = {
+    conversationID: ChatConstants.keyToConversationID(conversationIDKey),
+    tlfName: teamname,
+    tlfPublic: false,
+    headline: newTopic,
+    identifyBehavior: RpcTypes.TlfKeysTLFIdentifyBehavior.chatGui,
+  }
+
+  // TODO figure out a better waiting solution.
+  yield all([
+    put(setRouteState(path, s => s.update('waitingForSave', 0, n => n + 1))),
```

Comment:
I'm not in love with this, but it was the fastest thing I came up with. Open to suggestions